pkgs/tools/typesetting/lowdown/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchurl, which }:
+
+stdenv.mkDerivation rec {
+  pname = "lowdown";
+  version = "0.7.2";
+
+  outputs = [ "out" "lib" "dev" "man" "devman" ];
+
+  src = fetchurl {
+    url = "https://kristaps.bsd.lv/lowdown/snapshots/lowdown-${version}.tar.gz";
+    sha512 = "3ks1jfw4rjm0qb87ask7wx0xx1grxhbpg53r86q74zhsiqqi6xiza2czg75mydmgic1nr9ny43d5p44sl8ihhja9kwdx230nblx1176";
+  };
+
+  nativeBuildInputs = [ which ];
+
+  configurePhase = ''
+    ./configure PREFIX= \
+                BINDIR=''${!outputBin}/bin \
+                MANDIR=''${!outputMan}/share/man \
+                INCLUDEDIR="''${!outputDev}/include" \
+                LIBDIR="''${!outputLib}/lib"
+  '';
+
+  postInstall = ''
+    mkdir -p ''${!outputDevman}/share/man
+    mv ''${!outputMan}/share/man/man3 ''${!outputDevman}/share/man/
```

Comment:
I wouldn’t bother about `devman`, it’s just more overhead for little gain (plus it’s confusing when the manpage is missing).

Reply:
`nixpkgs` manual says "`$outputMan` is for man pages (except for section 3)."

Maybe it's clearer, if we install user man pages to `out` directly, `man3` to `devMan` and drop the `man` output altogether?

Reply:
I used `outputs = [ "out" "dev" ]`, any other outputs are overkill given their sizes. For example, `lib` only contains a static library so it's not useful to have it separate from `dev`.

Reply:
Seems sane, but this way all manpages end up in `out`, since a fixup hook moves them there. Might be confusing if the dev docs are shipped with the executables?